.htaccess.dist

```diff
@@ -13,7 +13,7 @@
     #RewriteRule ^ https://%{HTTP_HOST}%{REQUEST_URI} [L,R=301]
 
     # prevent httpd from serving dotfiles (.htaccess, .svn, .git, etc.) - except let's encrypt challenge
-    RedirectMatch 403 ^/?\.(?!/well-known/acme-challenge/[\w-]{43}$)
+    RedirectMatch 403 ^/?\.(?!well-known/acme-challenge/[\w-]{43}$)
```

Comment:
In that case this should look more like this `RedirectMatch 403 ^(?!/\.well-known/acme-challenge/[\w-]{43}$)`. :thinking: